modules/images-configuration-blocked.adoc

```diff
@@ -49,7 +49,7 @@ status:
 Either the `blockedRegistries` registry or the `allowedRegistries` registry can be set, but not both.
 ====
 +
-The Machine Config Operator (MCO) watches the `image.config.openshift.io/cluster` CR for any changes to registries and reboots the nodes when it detects changes. Changes to the blocked registries appear in the `/etc/containers/registries.conf` file on each node.
+The Machine Config Operator (MCO) watches changes to the registries, then drains and uncordons the nodes when it detects changes. After the nodes return to the `Ready` state, changes to the blocked registries appear in the `/etc/containers/registries.conf` file on each node.
```

Comment:
This is an editorial change for 4.7 MCO change previously reviewed. No need for QE review.